telegrambots-abilities/src/main/java/org/telegram/abilitybots/api/bot/BaseAbilityBot.java

```diff
@@ -348,18 +348,27 @@ private void registerAbilities() {
                     .flatMap(ext -> stream(ext.getClass().getMethods())
                             .filter(checkReturnType(Reply.class))
                             .map(returnReply(ext)))
-                            .flatMap(Reply::stream);
+                    .flatMap(Reply::stream);
+
+            Stream<Reply> extensionCollectionReplies = extensions.stream()
+                    .flatMap(extension -> stream(extension.getClass().getMethods())
+                            .filter(checkReturnType(ReplyCollection.class))
+                            .map(returnReplyCollection(extension))
+                            .map(ReplyCollection::getReplies))
```

Comment:
I'd rather we add a `.stream` method to the ReplyCollection class and simply flatmap into it. `getReplies` can stay tho.